package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "electron-reload",
-  "version": "1.5.0",
+  "version": "1.6.0",
   "description": "Simplest way to reload an electron app on file changes!",
   "main": "main.js",
+  "types": "main.d.ts",
   "scripts": {
-    "lint": "standard",
-    "lint:fix": "standard --fix"
+    "lint": "./node_modules/.bin/standard",
+    "lint:fix": "./node_modules/.bin/standard --fix",
+    "build": "./node_modules/.bin/rimraf main.d.ts && npm run lint:fix && ./node_modules/.bin/tsc",
```

Comment:
Remove `rimraf` please (see bellow). I suppose `tsc` would overwrite the file and if not there is surely a flag to do so.

Reply:
Actually tsc will not overwrite the `.d.ts` as seen in this discussion https://github.com/Microsoft/TypeScript/issues/16749 . That is why I think he added the rimraf commands. 

@yan-foto you can check my other PR for just adding declaration. It is simple and easy to implement. Also this PR tackles 2 issue. We can focus on the other issue with this PR and resolve the issue for typescript in my PR #84 .

Reply:
I got it to work without rimraf by adding the .d.ts to the `exclude` rules

Reply:
I think you guys should work together. As might have already figured out, I am running very low on spare time so it's not feasible for me to attend similar PRs at the same time.

Reply:
@yan-foto  Is the anything else you want changed? If so, @Mackignacio do you want me to add you as a collaborator for my branch? I need to have #61 fixed so I can code my "main" file in typescript, then point the "mainFile" to my compiled .js file

Reply:
@josheleonard do you still need help for issue #61?

Reply:
@Mackignacio #61 should be fixed, feel free to commit any changes you feel are needed.